services/notify/test/helper.js

```diff
@@ -131,6 +131,30 @@ exports.withSES = (mock, skipping) => {
           TopicArn: snsArn,
           Endpoint: emailAttr.QueueArn,
         }).promise();
+
+        const Policy = {
+          Version: "2012-10-17",
+          Statement: [
+            {
+              Sid: "Sid1573761323466",
+              Effect: "Allow",
+              Principal: {AWS: "*"},
+              Action: "SQS:SendMessage",
+              Resource: emailAttr.QueueArn,
+              Condition: {
+                ArnEquals: {
+                  "aws:SourceArn": snsArn,
+                },
+              },
+            },
+          ],
+        };
+        await sns.setQueueAttributes({
+          QueueUrl: emailSQSQueue,
+          Attributes: {
+            Policy,
+          },
+        }).promise();
```

Comment:
What does this do?

Reply:
Makes it work? :)

I'll add a comment..